### PR TITLE
Sweet, sweet sorting

### PR DESCRIPTION
**Pull request by kurtmckee**
As discussed: a sorted `__all__`.

### CODE REVIEW

File: src/globus_sdk/_generate_init.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 
 HERE = pathlib.Path(__file__).parent
 
-FIXED_PREAMBLE = """\
+FIXED_PREAMBLE = f"""\
 # isort:skip_file
 # fmt:off
 #
-# this __init__.py file is generated by __generate_init.py
+# this __init__.py file is generated by {pathlib.Path(__file__).name}
```
Comment (sirosen) [inline src/globus_sdk/_generate_init.py:14]: Hah! Cute; I like it. 😄

**Review by sirosen — APPROVED** [review]
Who _doesn't_ like sorted data? Merging posthaste!

**Comment by sirosen** [comment]
Oh, also, I believe I only promised the 🚀 if you made the output `black`-friendly, but this still merits a rocket.

**Comment by kurtmckee** [comment]
Ooh, I didn't realize that the file was excluded from black -- I ran the black pre-commit hook against all files, got a success message, and called it good. Next time!

**Pull request merged**